packages/@okta/vuepress-theme-prose/global-components/ErrorCodes.vue

```diff
@@ -41,10 +41,8 @@
     <div id="error-code-count">
       Found <b>{{ resultCount }}</b> matches
     </div>
-    <div
+    <template
       v-for="oktaError in filteredErrorCodes"
-      :key="oktaError.errorCode"
-      class="error-code"
```

Comment:
This "v-for" wraps every error in a separate div - data on the page is organized. How does removing this wrapper help?